units/Power_Generator.cfg

```diff
@@ -24,7 +24,7 @@
     [/resistance]
     movement=0
     experience=150
-    level=3
+    level=8
```

Comment:
Did you really intend to change a unit that only appears in chapter 9?

Reply:
They are much harder to kill now, so the player deserves more XP.
There's five of them in scenarios5/27_Power_Station.cfg

Reply:
Aha, okay.